### ISSUE TITLE
test code

### ISSUE THREAD

**Issue by Yarong-zhang** [TreB1eN/Pytorch0.4.1_Openpose#14]
Hello, do you have the code about the test set with own trained model？

**Comment by neverstoplearn** [comment]
请问你解决了吗？谢谢。
